y_lb.libraries.yml

```diff
@@ -45,10 +45,10 @@ search:
       assets/css/search.css: { preprocess: true }
 
 bs_icons:
-  version: 1.10.3
+  version: 1.11.0
   css:
     theme:
-      //cdn.jsdelivr.net/npm/bootstrap-icons@1.10.3/font/bootstrap-icons.css: { external: true }
+      //cdn.jsdelivr.net/npm/bootstrap-icons@1.11.0/font/bootstrap-icons.css: { external: true }
```

Comment:
Instead of including pretty large css everywhere we can have own CSS with just one twitter-x 
```css
@font-face {
  font-display: block;
  font-family: "bootstrap-icons";
  src: url("./fonts/bootstrap-icons.woff2?1bb88866b4085542c8ed5fb61b9393dd") format("woff2"),
url("./fonts/bootstrap-icons.woff?1bb88866b4085542c8ed5fb61b9393dd") format("woff");
}
.bi::before,
[class^="bi-"]::before,
[class*=" bi-"]::before {
  display: inline-block;
  font-family: bootstrap-icons !important;
  font-style: normal;
  font-weight: normal !important;
  font-variant: normal;
  text-transform: none;
  line-height: 1;
  vertical-align: -.125em;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}
.bi-twitter-x::before { content: "\f8db"; }

```